README.md

```diff
@@ -48,6 +48,12 @@ scoop install deno
 brew install deno
 ```
 
+## Environment Variables
+
+- `DENO_INSTALL` - The directory in which to install Deno. This defaults to `$HOME/.deno`.
+  Not yet supported in the PowerShell installer (#76).
```

Comment:
Can you give a concrete example of usage here?

Reply:
👍